website/src/css/custom.css

```diff
@@ -107,3 +107,65 @@ svg.lightToggleIcon_node_modules-\@docusaurus-theme-classic-lib-next-theme-Color
 li.footer__item a svg {
   display: inline;
 }
+
+/* mo changes 20 oct 2022 */
+
+.alert {
+  max-width: 1500px;
+  margin: 2em auto;
+}
+
+html[data-theme='dark'] .navbar {
+  background-color: #1b1b1d !important;
+}
+
+.navbar,
+.title-font,
+.text-base {
+  font-family: 'Montserrat' !important;
```

Comment:
FYI we have sonarcloud check that is failing with non-default font https://sonarcloud.io/organizations/containers/rules?open=css%3AS4649&rule_key=css%3AS4649

Reply:
That's a really good rule of thumb, just pushed a commit to fix this